package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "neutrino-middleware-styleguidist",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
Could you remove the versioning from this commit? I will take care of bumping the version when I go to do the release.